OpenRA.Mods.Common/Traits/ProductionQueueFromSelection.cs

```diff
@@ -42,6 +42,7 @@ public ProductionQueueFromSelection(World world, ProductionQueueFromSelectionInf
 
 		void INotifySelection.SelectionChanged()
 		{
+			// TODO: Should this extend to controlling allied units?
```

Comment:
In my option this is a question beyond this PR. I would remove the comment from this PR.